docs/source/operations/upgrade_fabricnetwork.md

```diff
@@ -0,0 +1,26 @@
+<a name = "upgrading-fabric"></a>
+# Upgrading Hyperledger Fabric version
+
+- [Pre-requisites](#pre_req)
+- [Modifying image versions](#modify_image_version)
+
+<a name = "pre_req"></a>
+## Pre-requisites
+Hyperledger Fabric image versions, which are compatible with the target fabric version need to be known. Here's the list of the images for which the versions should be known:  
+kafka image-- hyperledger/fabric-kafka:`targer_version`  
```

Comment:
change this to a table, saying "For example, for Fabric v1.4.4, these are the image tags of the supporting docker images"